test/jellyfish_web/controllers/room_controller_test.exs

```diff
@@ -118,22 +118,6 @@ defmodule JellyfishWeb.RoomControllerTest do
                "Cannot add room with id \"#{room_id}\" - room already exists"
     end
 
-    test "renders room when max_peers isn't present", %{conn: conn} do
-      conn = post(conn, ~p"/room")
-      assert %{"id" => id} = json_response(conn, :created)["data"]["room"]
-
-      conn = get(conn, ~p"/room/#{id}")
-      response = json_response(conn, :ok)
-      assert_response_schema(response, "RoomDetailsResponse", @schema)
-
-      assert %{
-               "id" => ^id,
-               "config" => %{},
-               "components" => [],
-               "peers" => []
-             } = response["data"]
-    end
```

Comment:
Why did you remove this test?

Reply:
It doesn't add anything of value, the checks performed here are already done in "renders room when data is valid" and "renders room when data is valid, custom room_id"

Reply:
Practically speaking, yes; philosophically, no. It is not the responsibility of these tests to verify that action X occurs, especially if it's not mentioned in their names. However, it is easy to accidentally refactor these tests and unintentionally remove coverage for this edge case. If you insist on removing this test, I suggest adding a note to one of the other tests indicating that it also covers the situation previously handled by the removed test.

Reply:
Practically speaking, yes; philosophically, no. It is not the responsibility of these tests to verify that action X occurs, especially if it's not mentioned in their names. However, it is easy to accidentally refactor these tests and unintentionally remove coverage for this edge case. If you insist on removing this test, I suggest adding a note to one of the other tests indicating that it also covers the situation previously handled by the removed test or modifying their names.

Reply:
Updated name of test